canonical-websites/www.ubuntu.com
# 1621 download alternative downloads

## Done

* updated alternative downloads - http://0.0.0.0:8001/download/alternative-downloads
* fixed contextual footers to use p-link--external, instead of external and added equal height to each section so the dividers look even

## QA

- Check out this feature branch or the [demo](http://www.ubuntu.com-1621-download-alternative-downloads.demo.haus/download/alternative-downloads)
- Run the site using the command `./run serve`
- look at the above page and see if it looks ok
- fixed external link class and made equal heights

## Issue / Card

Fixes #1621
